### ISSUE TITLE
Add support for crawler_name in aws_glue_trigger

### ISSUE THREAD

**Issue by kurtmc** [terraform-providers/terraform-provider-aws#9542]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Description

As of this release of the go sdk https://github.com/aws/aws-sdk-go/blob/master/CHANGELOG.md#release-v1205-2019-06-20 you have been able to set glue triggers for either jobs or crawlers. I think we can just add an extra field to the actions block for aws_glue_trigger to support triggering crawlers.

API reference for action struct: https://docs.aws.amazon.com/sdk-for-go/api/service/glue/#Action

### New or Affected Resource(s)

* aws_glue_trigger

### Potential Terraform Configuration

```hcl
resource "aws_glue_trigger" "example" {
  name = "example"
  type = "CONDITIONAL"

  actions {
    crawler_name = "${aws_glue_crawler.example1.name}"
  }

  predicate {
    conditions {
      crawler_name = "${aws_glue_crawler.example2.name}"
      state    = "SUCCEEDED"
    }
  }
}

```

**Comment by anateus** [comment]
Just noticed that in the description the you're using the `state` field, rather than `crawl_state`. Is that just an oversight or an intentional preference for normalizing the two fields? I've implemented it to be one to one with the specs, though can see some ergonomic benefit to having logic that translates it into `CrawlState` if `crawler_name` is used rather than `job_name.

**Comment by kurtmc** [comment]
@anateus That's a mistake. I would prefer it to more closely match the sdk (https://docs.aws.amazon.com/sdk-for-go/api/service/glue/#Condition).

**Comment by anateus** [comment]
Great! In that case, just waiting for review/approval on my PR :)

**Comment by bflad** [comment]
Hi folks 👋 Support for the `crawler_name` argument in the `actions` configuration block and `crawler_name` and `crawl_state` arguments in the `predicate` `conditions` configuration block has been merged and will release with version 2.39.0 of the Terraform AWS Provider, by Thursday. Thanks to @anateus for the implementation. 👍
